### PR TITLE
Add tag workflow and readme

### PR DESCRIPTION
**Pull request by jencub**

### CODE REVIEW

File: values.yaml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 ---
 image:
   repository: gcr.io/planet-4-151612/robots
-  tag: v0.1
+  tag: latest
```
Comment (27Bslash6) [inline values.yaml]: Using tag:latest for docker images won't necessarily force a redeploy when the image changes, better to pin to actual releases.

Comment (27Bslash6) [inline values.yaml]: The problem with changing the tag to :latest here is that helm won't necessarily force a redeploy when the image updates - there's no field to diff against.  Best to pin to actual release versions.

Also, see https://stackoverflow.com/questions/33112789/how-do-i-force-kubernetes-to-re-pull-an-image - unless `imagePullPolicy` is `Always` (it's never Always), nodes will use *whatever* version of :latest they have in the node-local container cache.

**Review by 27Bslash6 — CHANGES_REQUESTED** [review]

File: values.yaml (hunk 1 above)
Comment (jencub) [reply values.yaml]: version applied as suggested and readme update to reflect need to tag first.

**Review by 27Bslash6 — APPROVED** [review]
👏

**Pull request merged**